audiomate/logutil.py

```diff
@@ -1,66 +1,18 @@
-""" Logging functionaly to use within audiomate. """
+""" Logging functionality to use within audiomate. """
 
 import logging
-import time
 
-
-PROGRESS_LOGGER_DELAY_SECONDS = 5 * 60
-PROGRESS_LOGGER_LEVEL = logging.INFO
+import tqdm
```

Comment:
This undos our effort of not having tqdm in the library.
If you want to use tqdm, you have to wait for #116 or implement it yourself.

Reply:
What does speak agains tqdm? Especially if you can exchange it easily with another library in my implementation.

It provides important information in long running tasks, for example that I have to wait 33h until the german CV is converted correctly. I'm not sure why to use a custom implementation for this.

Reply:
We had that before: https://github.com/ynop/audiomate/pull/112#pullrequestreview-383421507

Supporting tqdm is okay, but this should be an opt-in feature. We want to keep audiomate fit for a broad range of tasks/situations.

Reply:
What about and opt-out feature,? If someone really don't wants it he can remove it. I still think a progress bar is very important for the long running tasks. What does speak against tqdm installed by default? You're already installing a lot of big libraries like scipy, pandas or librosa.